app/src/main/java/com/example/footballmanager/data/network/RetrievingDataState.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.footballmanager.data.network
+
+import com.example.footballmanager.data.entities.Match
+
+sealed interface RetrievingDataState {
+    data class Success(val matches: List<Match>, val cached: Boolean) : RetrievingDataState
```

Comment:
try to keep code formatting consistent - each parameter in a new line, as in the Error data class